luigi/parameter.py

```diff
@@ -139,7 +159,10 @@ def __init__(self, default=_no_value, is_global=False, significant=True, descrip
                                 ``positional=False`` for abstract base classes and similar cases.
         :param bool always_in_help: For the --help option in the command line
                                     parsing. Set true to always show in --help.
-        :param str batch_method: How multiple
+        :param BatchAggregation batch_method: How multiple values of this argument are combined when
+                                              batching in the scheduler. See BatchAggregation for
+                                              details of what each one means. Default is None,
+                                              meaning that this parameter cannot be aggregated.
```

Comment:
The COMMA_LIST refers to how we combine serializations. If we combined them as an actual list, it would break deserialization. It's up to the user to define the parameters to deserialize into a list if that's what they want. If we build in the list serialization, it would be very similar to re-implementing list parameters, which you were big on removing.

What we could do instead is provide a wrapper to translate between lists and commas. So instead of  `IntParameter(batch_method=COMMA_LIST)` you would do `batched_list(IntParameter())`. That way we don't need to build any special-case code in the Parameter class just for COMMA_LIST.